lib/code_corps/stripe_service/validators/project_can_enable_donations.ex

```diff
@@ -28,5 +32,16 @@ defmodule CodeCorps.StripeService.Validators.ProjectCanEnableDonations do
     donation_goals: [_h | _t],
     organization: %Organization{stripe_connect_account: %StripeConnectAccount{charges_enabled: true}}
   } = project), do: {:ok, project}
+
+  defp do_validate(%Project{
+    donation_goals: [_h | _t],
```

Comment:
What's the purpose of this? I don't see donation goals being used. Is at least one donation goal required?